src/test/java/javafxlibrary/utils/HelperFunctionsTests/GetMouseButtonsTest.java

```diff
@@ -50,7 +53,7 @@ public void getMouseButtons_MultipleValues() {
     public void getMouseButtons_InvalidValue() {
         thrown.expect(JavaFXLibraryNonFatalException.class);
         // thrown.expect(IllegalArgumentException.class);
-        thrown.expectMessage("\"HUGE_RED_ONE\" is not a valid MouseButton. Accepted values are: [NONE, PRIMARY, MIDDLE, SECONDARY]");
+        thrown.expectMessage("\"HUGE_RED_ONE\" is not a valid MouseButton. Accepted values are: " + Arrays.asList(MouseButton.values()));
```

Comment:
I had a mouse with back and forward buttons. Test failed since message had BACK and FORWARD values.